WordPress/Classes/ViewRelated/Developer/StoreSandboxSecretScreen.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+struct StoreSandboxSecretScreen: View {
+    private static let storeSandboxSecretKey = "store_sandbox"
+
+    @SwiftUI.Environment(\.presentationMode) var presentationMode
+    @State private var secret: String
+    private let cookieJar: CookieJar
+
+    var body: some View {
+        NavigationView {
+            VStack(alignment: .leading) {
+                Text("Enter the Store Sandbox Cookie Secret:")
+                TextField("Secret", text: $secret)
+                    .padding(EdgeInsets(top: 16, leading: 16, bottom: 16, trailing: 16))
+                    .border(Color.black)
+                Spacer()
+            }
+            .padding(EdgeInsets(top: 16, leading: 16, bottom: 16, trailing: 16))
+        }
+        .onDisappear() {
+            // This seems to be necessary do to an iOS bug where
+            // accessing presentationMode.wrappedValue crashes.
+            DispatchQueue.main.async {
+                if(self.presentationMode.wrappedValue.isPresented == false) {
+                    if let cookie = HTTPCookie(properties: [
```

Comment:
can we compact this into `if self.presentationMode.wrappedValue.isPresented == false, let cookie = HTTPCookie(...`?

Reply:
Done